pkgs/build-support/build-graalvm-native-image/default.nix

```diff
@@ -0,0 +1,53 @@
+{ lib, stdenv, graalvmCEPackages, glibcLocales }:
+
+{ name ? "${args.pname}-${args.version}"
+  # Final executable name
+, executable ? args.pname
+  # JAR used as input for GraalVM derivation, defaults to src
+, jar ? args.src
+, dontUnpack ? (jar == args.src)
+  # Extra arguments to be passed to the native-image
+, extraNativeImageBuildArgs ? [ ]
+  # XMX size of GraalVM during build
+, graalvmXmx ? "-J-Xmx6g"
+  # The GraalVM to use
+, graalvm ? graalvmCEPackages.graalvm11-ce
```

Comment:
Is there a reason for using GraalVM 11 instead of 17? Both are LTS, I believe.

Reply:
I could set this to 17, but them I would set need to set GraalVM 11 to all packages.

I don't think I ever saw a package building with GraalVM 17 instead of 11 yet.

Reply:
That's fair.